packages/core/src/components/checkbox/checkbox.stories.js

```diff
@@ -120,3 +121,11 @@ export const GroupWithParent = () => {
 }
 
 GroupWithParent.storyName = 'Group with a parent';
+
+export const WithHelperText = () => `
+    <div class="hds-checkbox">
+      <input type="checkbox" id="checkbox7" class="hds-checkbox__input" value="foo" />
+      ${getLabel('checkbox7')}
+      ${getHelperText()}
+    </div>
```

Comment:
I noticed that our HDS Core examples have some accessibility flaws. The helper-text should be linked with the input. Otherwise the screen readers read it separately and the connection to the input is lost. Often the helper text is extra info for the input. 
This can be done by adding aria-describedby attribute to input like in HDS React library. The attribute can hold multiple element ids, for example <input aria-describedby="helper-text-id error-text-id" type="text"/>

In here it could be like:
```suggestion
const helperTextId = 'helper-text-id';
const getHelperText = (text = 'Assistive text') => `<span id="${helperTextId}" class="hds-radio-button__helper-text">${text}</span>`;

export const WithHelperText = () => `
    <div class="hds-checkbox">
      <input type="checkbox" id="checkbox7" class="hds-checkbox__input" value="foo" aria-describedby="${helperTextId}" />
      ${getLabel('checkbox7')}
      ${getHelperText()}
    </div>
```